doc/_config.yml

```diff
@@ -0,0 +1,32 @@
+# Book settings
+# Learn more at https://jupyterbook.org/customize/config.html
+
+title: Visualization Curriculum
+author: UW Data
+logo: idl-logo.png
```

Comment:
What do you think about a more generic logo. 

Maybe start with this image from the notebooks

![image](https://user-images.githubusercontent.com/589034/108622211-821e9e00-7437-11eb-9f2d-e6b931661465.png)


[Open the Chart in the Vega Editor](https://vega.github.io/editor/#/url/vega-lite/N4Igxg9gdgZglgcxALlANzgUwO4tAZwBcAnCAa0xRBIEMp8AHG4zKQkAXwBoQaAPOPjwgExOABMAwhAA2EYlVr0mLNp24hxNQjWEBXYjKoALQoQb5kAeithxUAHQArfOMwy4aYg6iZCVqAYAWys0TAQaAFotHXw-fAABAEYrGJorIIgMTHxnfGhOHiDmMioAI2YQHlZIcTgoJFQQPmFCAE8GSmQQAEc9OkI4HUGwqpAy+uFivgn6FAAmAAYNeHdxKgBJAFkAEQAhAH0AJW16pB5+QRQoPRkZDTbWjq7e-rYh09GLhFFw7RfIHo1BcBEJkDc7hwNMZMIhTCgkgBWRY8AAk+DAMOKJjMFmsoT+DgQQ2MejKDjgECsGKx6TCEUiHkImFCABYHAAOBxJPIFDhAA)

Reply:
Sounds good! This is how it looks with what you posted and a few variants, let me know which one you prefer:

1.
![image](https://user-images.githubusercontent.com/4560057/108632116-1bdb5080-7422-11eb-9f17-3104f7b2b43d.png)

2.
![image](https://user-images.githubusercontent.com/4560057/108632141-36152e80-7422-11eb-96dc-565cb58e4bfa.png)

3.
![image](https://user-images.githubusercontent.com/4560057/108631923-3e209e80-7421-11eb-95c7-21ab3a5c72a3.png)

4.
![image](https://user-images.githubusercontent.com/4560057/108631959-5a244000-7421-11eb-9c3c-b9f1a7370801.png)

5.
![image](https://user-images.githubusercontent.com/4560057/108631970-63151180-7421-11eb-93f1-ddae9b14a827.png)

Reply:
I like 4. with the nod to Altair.

Reply:
Done! I added the VegaEditor URL as a comment in the config file in case anyone wants to modify it in the future.